cluster/migrations/000004_email-templates.up.sql

```diff
@@ -0,0 +1,29 @@
+-- This file is part of MinIO Kubernetes Cloud
+-- Copyright (c) 2019 MinIO, Inc.
+--
+-- This program is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU Affero General Public License as published by
+-- the Free Software Foundation, either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU Affero General Public License for more details.
+--
+-- You should have received a copy of the GNU Affero General Public License
+-- along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+CREATE TABLE email_templates
+(
+    id       VARCHAR(256) NOT NULL
```

Comment:
we should use either `id` type UUID or `id` type UUID and `name` as other column to be consistent

Reply:
the thing is that the template will have a unique key based on name, if we rely on `id` it will be an unused field since we will only reference the template by their name, therefore the name should be it's primary key

Reply:
we should then reference it by id and used the name just when querying it  similiar to what we do on ServiceAccounts and Permissions